### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: content/lxd/news.ja/lxd-2.0.0.beta1.yaml
Comment (ru-fu) [inline content/lxd/news.ja/lxd-2.0.0.beta1.yaml:23]: Looks like this was spelled wrong on purpose.

File: content/lxd/news/lxd-2.0.0.beta1.yaml
Comment (ru-fu) [inline content/lxd/news/lxd-2.0.0.beta1.yaml:22]: This shouldn't be changed.

**Review by ru-fu — COMMENTED** [review]
Thanks! I spotted one case that shouldn't be changed though, because it's pointing out a fixed typo.

File: content/lxd/news.ja/lxd-2.0.0.beta1.yaml
Comment (kianmeng) [reply content/lxd/news.ja/lxd-2.0.0.beta1.yaml]: > Looks like this was spelled wrong on purpose.

@ru-fu Done.

File: content/lxd/news/lxd-2.0.0.beta1.yaml
Comment (kianmeng) [reply content/lxd/news/lxd-2.0.0.beta1.yaml]: > This shouldn't be changed.

@ru-fu Reverted.

**Review by ru-fu — APPROVED** [review]

**Comment by stgraber** [comment]
@kianmeng hey there, before we can merge your fix, we'd need the commit message to include the sign-off line (see contributing.md).

This is usually done with `git commit --amend -s` and then followed with `git push --force`.
It takes the form of a line that looks like `Signed-off-by: Firstname Lastname <email>`

**Comment by kianmeng** [comment]
> @kianmeng hey there, before we can merge your fix, we'd need the commit message to include the sign-off line (see contributing.md).
> 
> This is usually done with `git commit --amend -s` and then followed with `git push --force`.
> It takes the form of a line that looks like `Signed-off-by: Firstname Lastname <email>`

@stgraber Done and thanks for the instruction.

File: static/js/term.js
Comment (stgraber) [inline static/js/term.js:3513]: I'd drop the changes to this file as it's a verbatim copy from an external project and altering it will make it much harder for us to keep track of it.

**Comment by stgraber** [comment]
Thanks, just need to drop the change to the term.js

File: static/js/term.js
Comment (kianmeng) [reply static/js/term.js]: > I'd drop the changes to this file as it's a verbatim copy from an external project and altering it will make it much harder for us to keep track of it.

@stgraber Got it. Reverted.

**Review by ru-fu — APPROVED** [review]

**Pull request merged**
